02-sea-level.Rmd

```diff
@@ -0,0 +1,185 @@
+```{r echo=FALSE}
+yml_content <- yaml::read_yaml("chapterauthors.yml")
+author <- yml_content[["lab2shortname"]][["author"]]
+```
+# Mapping Sea Level Rise with Digitial Elevation Models {#SeaLevelRise}
+
+Written by
+```{r results='asis', echo=FALSE}
+cat(author)
+```
+
+## Lab Overview {.unnumbered}
```

Comment:
I think we can incorporate some simple hydrological tools in this lab in another task. I propose the following sequence:

1. Flow direction
2. Flow accumulation
3. Reclassify #2 using a threshold
4. Convert #4 to polylines to derive streams and rivers (Stream to Feature)
5. Derive watersheds

We can incorporate these layers in the map making task.